src/System.Management.Automation/resources/ParameterBinderStrings.resx

```diff
@@ -142,7 +142,7 @@
     <value>Cannot bind positional parameters because no names were given.</value>
   </data>
   <data name="AmbiguousParameterSet" xml:space="preserve">
-    <value>Parameter set cannot be resolved using the specified named parameters.</value>
+    <value>Parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided.</value>
```

Comment:
Not a grammar expert, but the last part of this sentence reads weird to me.  I know that `were` is used with plural nouns, so you would think that because `parameters` is used, it should be `were`, but I think the noun here is actually `an insufficient number of parameters` which is singular (the `an` gives it away) sort of like `...or an error was provided.`.  So it seems that it would be `...parameters was provided.`.

This is not blocking feedback.

Reply:
@SteveL-MSFT This is one of those weird English rules. 'a number of X-es` are plural while `the number of X-es` is singular. In this case we are using `an insufficient number of parameters` and is the plural form of the phrase. https://en.oxforddictionaries.com/explore/number-of-people-is-or-are

Reply:
:-) Now I know who to ask to approve error messages.

Reply:
@markekraus Could you please look test failures? It seems HttpListener can not start on 8082 port  in `Describing Web cmdlets tests using the cmdlet's aliases`. Also these tests is "alias" tests - I think we shiould remove its (we have separate test set for aliases).

Reply:
@iSazonov There is an open issue for tracking the HttpListener issue #5464.